mathesar_ui/src/systems/table-view/actions-pane/record-operations/filter/FilterEntries.svelte

```diff
@@ -28,6 +28,10 @@
     columns={$processedColumns}
     getColumnLabel={(column) =>
       $processedColumns.get(column.id)?.column.name ?? ''}
+    getColumnConstraintType={(column) => {
+      const linkFkType = $processedColumns.get(column.id)?.linkFk?.type;
+      return linkFkType ? [linkFkType] : undefined;
```

Comment:
Hi @seancolsen, while looking for a method to get constraint type from processedColumn. I noticed that in https://github.com/centerofci/mathesar/blob/develop/mathesar_ui/src/components/column/ProcessedColumnName.svelte#L17, we get constraint type using linkFk.type. I would like to know, if there's a better appoach to compute constraint type.